packages/ui/src/components/w3m-spinner/index.ts

```diff
@@ -1,17 +1,28 @@
 import { html, LitElement } from 'lit'
-import { customElement } from 'lit/decorators.js'
+import { customElement, property } from 'lit/decorators.js'
 import { ThemeUtil } from '../../utils/ThemeUtil'
 import styles from './styles.css'
 
 @customElement('w3m-spinner')
 export class W3mSpinner extends LitElement {
   public static styles = [ThemeUtil.globalCss, styles]
 
+  @property() public color = 'accent'
+  @property({ type: Boolean }) public small = false
```

Comment:
Maybe it would be more versatile to make this `@property() public size = 24` and allow more granular overrides with numbers?

Alternatively, it could be something like `@property() public size: 'small' | 'medium' = 'medium'`

Both of these allow us to set extra sizes in the future if needed.